src/ContextualPopup/ContextualPopup.js

```diff
@@ -538,18 +545,37 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	showHideScrim: function (inShow) {
+	updateScrim: function (old, value, source) {
+		if (this._updateScrimMutex) return;
```

Comment:
Hi @aarontam, 
I think I can have two choices for preventing multiple re-entrance.

1) Use mutex "this._updateScrimMutex"
2) Use assign "this.modal = this.spotlightModal;"

Which one do you prefer?